docs/reference/release-notes/highlights-7.2.0.asciidoc

```diff
@@ -21,3 +21,30 @@ summarize your data and store it in a new index. Alternatively, you can use
 the transforms.
 
 // end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+==== Closed indices are now replicated
+
+Elasticsearch 7.2.0 brings better support for closed indices by allowing
+shards of closed indices to be replicated. As soon as an index is closed,
+Elasticsearch takes care of safely tearing down the "opened" shards before
+reinitializing them as "closed" shards, which require much less resources
+while still providing the ability for shards to be promoted as primaries or
+to be recovered from peer. The data is also automatically replicated by the
+cluster to ensure that enough shard copies are safely kept around at all
+times (configurable with `index.number_of_replicas`).
+
+In addition to that, it is now possible to snapshot closed indices using
+the <<modules-snapshots,Snapshot/Restore API>>. To include a closed index
+when creating a snapshot on Elasticsearch 7.2+, the `expand_wildcards`
+parameter must be explicitly set to either `all` or `closed` .
+
+To snapshot closed indices,
+setting the expand_wildcards option either to all or closed (see https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-index.html).
```

Comment:
this paragraph is a clone of the previous one? Just delete it?

Reply:
Sorry, I did not notice it